packages/web-components/package.json

```diff
@@ -90,7 +91,7 @@
     "ts-loader": "^7.0.2",
     "ts-node": "^8.0.0",
     "tsconfig-paths": "^3.9.0",
-    "typescript": "3.9.7",
+    "typescript": "^3.9.7",
```

Comment:
This was locked intentionally, if we aren't updating this package as part of this PR is there a reason this needs to change?

Reply:
what @chrisdholt said. can we please revert this change? we want to get rid of no exact versions in general. ty